assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app/story';
+import DraggablePage from '../draggablePage';
+import Rectangle from './rectangle.svg';
+
+const PAGE_WIDTH = 90;
+const PAGE_HEIGHT = 160;
+
+const Wrapper = styled.div`
+	position: relative;
+	display: grid;
+	grid-template-columns: ${ ( { scale } ) => `repeat(auto-fit, minmax(${ scale * PAGE_WIDTH }px, max-content))` };
+	grid-gap: 20px;
+	justify-content: center;
+	justify-items: center;
+    align-items: center;
+`;
+
+const RangeInputWrapper = styled.div`
+	display: flex;
+	width: 440px;
+	margin: 0 auto 75px;
+`;
+
+const rangeThumb = css`
+	width: 28px;
+	height: 28px;
+	border-radius: 100%;
+	background: ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const rangeTrack = css`
+	background: rgba(255, 255, 255, 0.2);
+	border-color: transparent;
+	color: transparent;
+	width: 100%;
+	height: 4px;
+`;
+
+// Lots of repetition to avoid browsers dropping unknown selectors.
+const RangeInput = styled.input.attrs( () => ( {
+	type: 'range',
+} ) )`
+	-webkit-appearance: none;
+	background: transparent;
+	display: block;
+	width: 360px;
+	margin: 0 auto;
+
+	&::-webkit-slider-thumb {
+		${ rangeThumb }
+		-webkit-appearance: none;
+		margin-top: -12px;
+	}
+
+	&::-moz-range-thumb {
+		${ rangeThumb }
+	}
+
+	&::-ms-thumb {
+		${ rangeThumb }
+	}
+
+	&::-webkit-slider-runnable-track {
+		${ rangeTrack }
+	}
+
+	&::-moz-range-track {
+		${ rangeTrack }
+	}
+
+	&::-ms-track {
+		${ rangeTrack }
+	}
+`;
+
+const RectangleIcon = styled.div`
+	margin-top: ${ ( { isLarge } ) => isLarge ? '0' : '6px' };
+
+	svg {
+		width: ${ ( { isLarge } ) => isLarge ? '20px' : '12px' };
+		height: ${ ( { isLarge } ) => isLarge ? '32px' : '20px' };
+	}
+`;
+
+function RangeControl( { value, onChange } ) {
+	return (
+		<RangeInputWrapper>
+			<RectangleIcon>
+				<Rectangle />
```

Comment:
Confirning just in case: These two rectangle svg-s shouldn't be interactive, right? (Should be "Unclickable" as they are now)

Reply:
It is unclear at the moment, just like the amount of steps in-between and their layout. Needs UX confirmation.

Reply:
Asked for clarification on the issue

Reply:
OK, we could def leave this for future improvements if it's the only blocker currently. AFAI can see, the design/descriptions in Figma don't show any hints on these being interactive at this moment, so ...

Reply:
SGTM 👍